lib/src/components/time_input/time_input.dart

```diff
@@ -220,9 +169,21 @@ class ZetaTimeInputState extends State<ZetaTimeInput> {
       context: context,
       initialTime: _value ?? TimeOfDay.now(),
       builder: (BuildContext context, Widget? child) {
-        return MediaQuery(
-          data: MediaQuery.of(context).copyWith(alwaysUse24HourFormat: !_use12Hr),
-          child: child!,
+        return Theme(
+          data: Theme.of(context).copyWith(
+            timePickerTheme: TimePickerThemeData(
```

Comment:
something not right here with text color when focused
![image](https://github.com/ZebraDevs/zeta_flutter/assets/23633655/3d1b14fa-b31e-4413-936d-0f139d637db1)